app/views/doctors/transcripts/show.haml

```diff
@@ -0,0 +1,13 @@
+%p
+  %b Description:
+  = @transcript.chat_room.urgent_request.description     
+%p
+  %b Patient:
+  = @transcript.chat_room.patient.name 
+%p
+  %b Transcript creation time:
+  = @transcript.updated_at
+%p
+  %b Chat:
+  
+= link_to 'Back', patient_transcripts_path(current_user.patient), class: 'btn btn-info'
```

Comment:
`patients_transcripts_path` in a doctors/index? Think you might want to use `doctor_transcripts_path`